### PR TITLE
improvement: Make dialogs look more like dialogs

### PR DESCRIPTION
**Pull request by lipis**
## Before

Screenshot 2020-12-29 at 4 10 19 PM

## After

Screenshot 2020-12-29 at 4 11 23 PM

-----

## TODO

- [x] Border
- [x] Border radius
- [x] Box shadow
- [x] Center aligned
- [x] Small Caps
- [x] Title background
- [x] Mobile view
- [x] Close button
- [x] Dark mode

### CODE REVIEW

**Comment by dwelle** [comment]
I'd maybe add it to Islands in general? E.g. the Stats dialog could use one.

Btw, why do we add `overflow-y: scroll` to modals by default? Kinda ugly:

image

**Comment by lipis** [comment]
> I'd maybe add it to Islands in general? E.g. the Stats island could use one.

Let's experiment with Islands in another PR... they have a shadow so they seem fine.

> Btw, why do we add `overflow-y: scroll` to modals by default? Kinda ugly:

Removed the scroll..

File: src/components/Modal.scss [hunk 1]
```diff
@@ -35,13 +35,19 @@
     transform: translateY(10px);
     animation: Modal__content_fade-in 0.1s ease-out 0.05s forwards;
     position: relative;
+    overflow-y: auto;
 
     // for modals, reset blurry bg
     background: var(--bg-color-island);
     backdrop-filter: none;
 
+    border: 1px solid var(--dialog-border);
```
Comment (j-f1) [inline src/components/Modal.scss:44]:
```suggestion
    border: 1.5px solid var(--dialog-border);
```

**Review by j-f1 — APPROVED** [review]

**Comment by dwelle** [comment]
Trying out box-shadow instead of border (as discussed in chat):

border:
image

shadow:

image

If the shadow is too big, we can decrease. IMO looks more discrete.

Anyway, the border was also causing some overflow/scrollbar issue on body for smaller viewports (we'd have to investigate).

**Comment by j-f1** [comment]
Needs some tweaks on mobile

**Comment by dwelle** [comment]
Let's drop the title background. It adds too many visual boxes. Without it looks better and cleaner:

image

image

**Comment by lipis** [comment]
Updated.

**Comment by j-f1** [comment]
Why the small caps for the title? I feel like the design on `master` for the title is mostly OK already (except maybe for centering the title)

**Comment by lipis** [comment]
> Why the small caps for the title? I feel like the design on `master` for the title is mostly OK already (except maybe for centering the title)

It made more sense with the title background.. and it was too big (h2)..

**Comment by dwelle** [comment]
I have a strange fetish for small caps titles as well, but what do I know.

@lipis btw, I meant for the box-shadow to replace the border. IMO the border is too strong and doesn't add much value on top of the shadow.

**Comment by lipis** [comment]
> @lipis btw, I meant for the box-shadow to replace the border. IMO the border is too strong and doesn't add much value on top of the shadow.

Yeah, I know.. I think it makes it sharper and the shadow is not that visible in the dark mode.. I tweaked the colors are bit.

**Review by ad1992 — APPROVED** [review]
Suggestion: I think this should be an improvement instead of a feature  as it makes the existing feature better.

**Comment by dwelle** [comment]
Ok, looks good!

**Pull request merged**